### PR TITLE
Add carbonapi_uuid to http response header

### LINKED ISSUE #36: Expose carbonapi_uuid in http response headers
Presenting carbonapi_uuid to a user with result of render will allow easier troubleshooting by user.
Suggested name is X-Carbonapi-uuid

### PR DESCRIPTION
**Pull request by azhiltsov**
Fixes #36

### CODE REVIEW

File: app/carbonapi/http_handlers.go [hunk 1]
```diff
@@ -127,8 +127,9 @@ func initHandlers(app *App) http.Handler {
 	return r
 }
 
-func writeResponse(w http.ResponseWriter, b []byte, format string, jsonp string) {
+func writeResponse(w http.ResponseWriter, b []byte, format string, jsonp string, ctx context.Context) {
 
+	w.Header().Set("X-carbonapi-uuid", util.GetUUID(ctx))
```
Comment (grzkv) [inline app/carbonapi/http_handlers.go]: I think, "X-Carbonapi-UUID" would be a more canonical name.

**Comment by grzkv** [comment]
It can be that the exact placing is not perfect, but this part of the code needs refactoring anyway. So, I think it's perfectly fine for now.

**Comment by azhiltsov** [comment]
@grzkv The way I passed a context, is it fine?
Do you have a suggestion how to improve it?

File: app/carbonapi/http_handlers.go (hunk 1 above)
Comment (azhiltsov) [reply app/carbonapi/http_handlers.go]: I agree

**Comment by grzkv** [comment]
> @grzkv The way I passed a context, is it fine?
> Do you have a suggestion how to improve it?

Just as I said in the previous comment. It looks perfectly fine to me. We will make the structure better during refactoring.

**Pull request merged**